### PR TITLE
Ui fixes

### PR DESCRIPTION
**Pull request by Claymanus**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,26 +5,32 @@ HTML_FILES = user_howto.html\
...
-	docker run --cpuset-cpus=0-2 --memory=2G --rm -p 5900:5900 -p 5432:5432 -p 8888:8888 -v /var/run/postgresql:/var/run/postgresql -v $$(pwd):/PDOauth -it magwas/edemotest:master /PDOauth/tools/script_from_outside
+	docker run --cpuset-cpus=0-1 --memory=2G --rm -p 5900:5900 -p 5432:5432 -p 8888:8888 -v /var/run/postgresql:/var/run/postgresql -v $$(pwd):/PDOauth -it magwas/edemotest:master /PDOauth/tools/script_from_outside
 
 %.json: %.po
 	./tools/po2json $< >$@
 
 install: static static/locale/hu.json
 
-checkall: install tests integrationtests end2endtest xmldoc
+checkall: uris_ install tests integrationtests end2endtest xmldoc
```
Comment (magwas) [inline Makefile]: uris_ should go to install, not checkall

Comment (magwas) [inline Makefile]: no, there will be a deploy target, with uris and install

Comment (Claymanus) [inline Makefile]: done

**Comment by magwas** [comment]
closing and opening a new one to trigger build.

**Review by venranbrewt90 — COMMENTED** [review]
### mark-ups are where?

**Pull request closed without merging**